wondrous-app/components/Common/TaskMint/TaskMintViewActions/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Grid } from '@mui/material';
+import { SnackbarAlertContext } from 'components/Common/SnackbarAlert';
+import { useContext } from 'react';
+import { LINK } from 'utils/constants';
+import OpenseaButton from '../OpenseaButton';
+import { CommunityShare } from '../Steps/SuccessMint';
+import { Wrapper } from './styles';
+
+const TaskViewMintActions = ({ tokenId, taskId }) => {
```

Comment:
maybe rename it to MintViewAndShareOptions?
naming is hard here, perhaps just add a comment explaining which component